docs/welcome/how-is-etherlink-governed.md

```diff
@@ -0,0 +1,66 @@
+This mechanism makes Etherlink self-amending and empowers Tezos bakers to govern Etherlink’s kernel upgrades, security updates, and sequencer operators.
+
+Etherlink has separate governance processes for the kernel, for security incidents, and for the Sequencer Committee.
+To ensure that decisions accurately reflect the consensus of the Etherlink community, all three governance processes are designed with the same robust safeguards.
+Like Tezos's governance process, Etherlink's governance process promotes transparency and fairness in decision-making.
+
+## Kernel governance
+
+The Etherlink kernel governance process is a streamlined version of the [Tezos governance and self-amendment process](https://docs.tezos.com/architecture/governance).
+It consists of three periods: a Proposal period and a Promotion period, which are supervised by Etherlink's kernel governance contract, and a Cooldown period, which is enforced by the Etherlink kernel itself.
+
+### 1. Proposal period
+
+During this period, Etherlink users have one week to submit and upvote proposals for new kernels.
+Any user can submit kernel upgrade proposals and any baker can upvote proposals, with the weight of their vote determined by the voting power of their baker account.
+
+At the end of the period, if a proposal has enough voting power to meet a certain percentage of the total voting power, it moves to the next phase.
+If no proposal gathers adequate support, a new Proposal period begins.
+
+### 2. Promotion period
+
+When a proposal moves to the Promotion period, bakers vote on whether to adopt the proposed kernel update.
+To pass, the proposal must meet both of these standards:
+
+- Quorum: A quorum of all of the voting power in the system must vote either "Yea", "Nay", or "Pass."
+
+- Supermajority: The total voting power of the Yea votes must reach a supermajority.
+
+If the proposal reaches both the quorum and supermajority requirements, it moves to the next period.
+In either case, a new Proposal period begins.
+
+### 3. Cooldown period
+
+The Cooldown period is a delay in the process that gives developers and bakers time to adapt their code and infrastructure to the new Etherlink kernel.
+This period lasts 24 hours, and at the end, Etherlink automatically enables the new kernel.
+
+This period can overlap with another Proposal period.
+
+### Alignment with Tezos layer 1 governance
+
+Etherlink kernel governance Proposal and Promotion periods require careful synchronization with layer 1 governance periods:
+
+- Etherlink Proposal and Promotion periods must happen within a single Tezos layer 1 governance period; they cannot overlap with two different layer 1 governance periods. This ensures that each delegate's voting power is the same throughout the period.
+- The beginning of any Proposal or Promotion period must be aligned with the beginning of a layer 1 governance period.
```

Comment:
The Etherlink governance periods must be syncronised with layer 1 governance periods in a way that the beginning of any layer 1 period is aligned with some Etherlink period (Proposal or Promotion).